src/codemods/__testfixtures__/block-to-width-100.output.tsx

```diff
@@ -2,7 +2,7 @@ import { Button } from '@freenow/wave';
 import React from 'react';
```

Comment:
It seems prettier is running against codemods fixtures and formatting them, this should not happen and we need to rollback these files (everything under /codemods/__testfixtures_). I've created PR #387 so that prettier ignores codemods fixtures from now on